skyportal_fink_client/skyportal_fink_client.py

```diff
@@ -0,0 +1,150 @@
+# coding: utf-8
+import os
+import yaml
+from fink_client.consumer import AlertConsumer
+from astropy.time import Time
+
+import utils.skyportal_api as skyportal_api
+
+# open yaml config file
+with open(
```

Comment:
If this block is meant to be re-used at some point, it would be better to make a function.

Reply:
You are right, we open the config file to read its content multiple times, i'll create a function dedicated to that.